RecFCCeeCalorimeter/src/components/AugmentClustersFCCee.cpp

```diff
@@ -752,34 +756,71 @@ StatusCode AugmentClustersFCCee::execute([[maybe_unused]] const EventContext &ev
               theta_p4 = 0;
             }
 
+            double weightLog_E_max = std::max(0., m_thetaRecalcLayerWeights[k][layer] + log(E_cell_Max[layer+startPositionToFill] / sumEnLayer[layer+startPositionToFill]));
+            double weightLog_m1 = std::max(0., m_thetaRecalcLayerWeights[k][layer] + log(E_m1 / sumEnLayer[layer+startPositionToFill]));
+            double weightLog_m2 = std::max(0., m_thetaRecalcLayerWeights[k][layer] + log(E_m2 / sumEnLayer[layer+startPositionToFill]));
+            double weightLog_m3 = std::max(0., m_thetaRecalcLayerWeights[k][layer] + log(E_m3 / sumEnLayer[layer+startPositionToFill]));
+            double weightLog_m4 = std::max(0., m_thetaRecalcLayerWeights[k][layer] + log(E_m4 / sumEnLayer[layer+startPositionToFill]));
+            double weightLog_p1 = std::max(0., m_thetaRecalcLayerWeights[k][layer] + log(E_p1 / sumEnLayer[layer+startPositionToFill]));
+            double weightLog_p2 = std::max(0., m_thetaRecalcLayerWeights[k][layer] + log(E_p2 / sumEnLayer[layer+startPositionToFill]));
+            double weightLog_p3 = std::max(0., m_thetaRecalcLayerWeights[k][layer] + log(E_p3 / sumEnLayer[layer+startPositionToFill]));
+            double weightLog_p4 = std::max(0., m_thetaRecalcLayerWeights[k][layer] + log(E_p4 / sumEnLayer[layer+startPositionToFill]));
+
             sum_E_3Bin = E_m1 + E_cell_Max[layer+startPositionToFill] + E_p1;
             sum_E_5Bin = sum_E_3Bin + E_m2 + E_p2;
             sum_E_7Bin = sum_E_5Bin + E_m3 + E_p3;
             sum_E_9Bin = sum_E_7Bin + E_m4 + E_p4;
+            sum_weightLog_3Bin = weightLog_E_max + weightLog_m1 + weightLog_p1;
+            sum_weightLog_5Bin = sum_weightLog_3Bin + weightLog_m2 + weightLog_p2;
+            sum_weightLog_7Bin = sum_weightLog_5Bin + weightLog_m3 + weightLog_p3;
+            sum_weightLog_9Bin = sum_weightLog_7Bin + weightLog_m4 + weightLog_p4;
```

Comment:
Same comment as for L667.
And the previous 4 lines (sum_E_3bin = ...) could go inside the else { .. }. block of L787